ios/app/brave_main_delegate.mm

```diff
@@ -52,15 +52,6 @@
 
 void BraveMainDelegate::BasicStartupComplete() {
   auto* command_line(base::CommandLine::ForCurrentProcess());
-
-  syncer::ModelTypeSet disabledTypes = syncer::ModelTypeSet(
-      // syncer::PASSWORDS,
-      syncer::PROXY_TABS, syncer::AUTOFILL,
-      // syncer::PREFERENCES,
-      syncer::READING_LIST, syncer::USER_CONSENTS);
-
-  command_line->RemoveSwitch(switches::kDisableSyncTypes);
-  command_line->AppendSwitchASCII(switches::kDisableSyncTypes, syncer::ModelTypeSetToString(disabledTypes));
```

Comment:
@mkarolin Is there something that replaces this? Want to make sure that iOS won't have all sync types enabled after this merges

Reply:
Ah sorry I missed the new file `ios_chrome_sync_client.mm`

Reply:
@kylehickinson I added an override `chromium_src/ios/chrome/browser/sync/ios_chrome_sync_client.mm` that adds those types directly. If you think we should bring back the switch as our own we can do that too. I haven't tested this change though, so hope it works 🤞

Reply:
@mkarolin I can build locally and run through a test at some point before we merge to ensure sync still functions correctly 🙂